### PR TITLE
fix bug: count(*) without from clause is error

### PR DESCRIPTION
**Pull request by ouyuanning**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #4798 

## What this PR does / why we need it:
fix bug: count(*) without from clause is error

### CODE REVIEW

**Review by fengttt — COMMENTED** [review]
This fix is wrong.    Original behavour is correct.

Just improve error message.

**Comment by fengttt** [comment]
> This fix is wrong. Original behavour is correct.
> 
> Just improve error message.

My bad.  This seems to be the right fix.

File: pkg/sql/plan/base_binder.go [hunk 1]
```diff
@@ -665,16 +665,18 @@ func (b *baseBinder) bindFuncExprImplByAstExpr(name string, astArgs []tree.Expr,
...
 			if nval.String() == "*" {
-				name = "starcount"
 				if len(b.ctx.bindings) == 0 || len(b.ctx.bindings[0].cols) == 0 {
-					return nil, errors.New("", "can not find any column when rewrite count(*) to starcount(col)")
-				}
-				var newCountCol *tree.UnresolvedName
-				newCountCol, err := tree.NewUnresolvedName(b.ctx.bindings[0].cols[0])
-				if err != nil {
-					return nil, err
+					// return nil, errors.New("", "can not find any column when rewrite count(*) to starcount(col)")
```
Comment (fengttt) [inline pkg/sql/plan/base_binder.go:669]: Please write a good comment,

**Review by aunjgr — APPROVED** [review]

File: pkg/sql/plan/base_binder.go (hunk 1 above)
Comment (ouyuanning) [reply pkg/sql/plan/base_binder.go]: done

**Review by aressu1985 — APPROVED** [review]

**Pull request merged**